### PR TITLE
chore: prune non-upgradeable network vats from production config

### PR DESCRIPTION
**Pull request by dckc**
closes: #7044

## Description

reorg network / IBC bootstrap to avoid including vat-network in production config

### Security Considerations

facilitates audit of production config

### Scaling, Documentation Considerations

_none AFAICT_

### Testing Considerations

TODO:
 - [ ] _finish WIP commits_
 - [ ] try the smoke tests

### CODE REVIEW

File: packages/vats/src/core/basic-behaviors.js [hunk 1]
```diff
@@ -317,14 +285,15 @@ export const addBankAssets = async ({
   bldIssuerKit.resolve(bldKit);
 
   const assetAdmin = E(agoricNamesAdmin).lookupAdmin('vbankAsset');
+  // XXX interposes bootstrap between agoricNames and vat-bank
```
Comment (michaelfig) [inline packages/vats/src/core/basic-behaviors.js:288]: @michaelfig: I'll be able to fix this one with the callback abstraction (a closureless bound method descriptor).

File: packages/vats/src/core/startNetIBC.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  },
+};
+
+export const getNetIBCManifest = ({ restoreRef }, { installKeys }) => {
+  return {
+    manifest: NET_MANIFEST,
+    installations: {
+      provisionPool: restoreRef(installKeys.provisionPool),
+      walletFactory: restoreRef(installKeys.walletFactory),
+    },
```
Comment (michaelfig) [inline packages/vats/src/core/startNetIBC.js]: I'm baffled as to why these installations are needed.  Was this a typo?

Comment (dckc) [reply packages/vats/src/core/startNetIBC.js]: yes, typo

File: packages/vats/src/core/basic-behaviors.js (hunk 1 above)
Comment (dckc) [reply packages/vats/src/core/basic-behaviors.js:288]: Seems to me that the simpler fix is to just pass `assetAdmin` and not bother with a separate `updater`. The only risk would be that vat-bank abuses `.reserve()`.

**Comment by dckc** [comment]
obsolete in favor of #7623

**Pull request closed without merging**